packages/components/organisms/f-checkout/src/services/experimentService.js

```diff
@@ -0,0 +1,24 @@
+// Can we have this in a public repo?
```

Comment:
Yes, this is all client-side anyway, if someone wanted to see this they could just look in the data layer.

Reply:
And if not, it's too late anyway 😅

Reply:
https://media.giphy.com/media/COYGe9rZvfiaQ/giphy.gif